### PR TITLE
fix(actix-web): Manually start runtime in actix-web example

### PR DESCRIPTION
**Pull request by loewenheim**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

This removes the use of `#[actix_web::main]` and correctly initializes Sentry before the async runtime is started.

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by Swatinem — APPROVED** [review]
you can land this as-is, or fix up the `actix::main` usage at the same time, your call :-)

**Review by Swatinem — APPROVED** [review]
lgtm. It might be worth calling out this edge case related to thread pools in a central case in addition to the actix example.

File: src/platforms/rust/guides/actix-web/index.mdx [hunk 1]
```diff
@@ -9,12 +9,15 @@ The `sentry-actix` crate adds a middleware for [`actix-web`](https://actix.rs/)
 
 To use this middleware, configure Sentry then add it to your actix web app as a middleware. Because actix is generally working with non sendable objects and is highly concurrent, this middleware creates a new hub per request. As a result, many of the Sentry integrations, such as breadcrumbs, do not work unless you bind the actix hub.
 
+NB: Macros like `#[tokio::main]` and `#[actix_web::main]` are not supported. The Sentry client must be initialized before the async runtime is started so that all threads are correctly connected to the Hub.
```
Comment (vivianyentran) [inline src/platforms/rust/guides/actix-web/index.mdx:12]: Are you using NB as in nota bene here? We typically avoid latin phrases/acronyms in docs as they don't translate well for non-native speakers. Maybe you can use a note here instead?

**Review by vivianyentran — APPROVED** [review]

File: src/platforms/rust/guides/actix-web/index.mdx (hunk 1 above)
Comment (loewenheim) [reply src/platforms/rust/guides/actix-web/index.mdx:12]: Yeah, that's what I was going for. I'll change it.

**Pull request merged**